data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_sweden.asset

```diff
@@ -4,14 +4,9 @@ local layer = {
   Identifier = "WAC_Sweden",
   Name = "WAC [Sweden]",
   Enabled = asset.enabled,
-  Description = [[Lunar Reconnaissance Orbiter Camera (LROC) Wide Angle Camera (WAC)
-    aboard the Lunar Reconnaissance Orbiter (LRO) has allowed the instrument team to
-    create a global mosaic comprised of over 15,000 images acquired between November 2009
-    and February 2011. The WAC maps the whole Moon in one month, however the solar
-    incidence angle at the equator changes about 28° from the beginning to the end of the
-    month. To even out the incidence angle variations (generally angles between 55-75
-    degrees), this morphology mosaic (at 643 nm), is comprised of data collected over
-    three periods (1/20/2010 to 1/28/2010, 5/30/2010 to 6/6/2010, 7/24/2010 to 7/31/2010).
+  Description = [[LLunar Reconnaissance Orbiter Camera (LROC) Wide Angle Camera (WAC) aboard the Lunar Reconnaissance Orbiter (LRO) has allowed the instrument team to create a global mosaic comprised of over 15,000 images acquired between November 2009 and February 2011. The WAC maps the whole Moon in one month, however the solar incidence angle at the equator changes about 28° from the beginning to the end of the month. To even out the incidence angle variations (generally angles between 55-75 degrees), this morphology mosaic (at 643 nm), is comprised of data collected over three periods (1/20/2010 to 1/28/2010, 5/30/2010 to 6/6/2010, 7/24/2010 to 7/31/2010).
```

Comment:
Extra L in Lunar, beginning of first sentence.

Reply:
Yea, the text is copied from https://astrogeology.usgs.gov/search/map/Moon/LRO/LROC_WAC/Lunar_LRO_LROC-WAC_Mosaic_global_100m_June2013.  It does say "(Description from URL)" but that line jumped one too far down